assets/js/blocks/cart-checkout/cart/block.js

```diff
@@ -1,13 +1,61 @@
 /**
  * External dependencies
  */
-import { Placeholder } from '@wordpress/components';
+import { BlockControls } from '@wordpress/block-editor';
+import { Toolbar } from '@wordpress/components';
+import { Fragment, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import TextToolbarButton from '@woocommerce/block-components/text-toolbar-button';
+
+/**
+ * Internal dependencies
+ */
+import FullCart from './components/full-cart';
+import EmptyCart from './components/empty-cart';
 
 /**
  * Component to handle edit mode of "Cart Block".
  */
 const Cart = () => {
-	return <Placeholder>Cart block coming soon…</Placeholder>;
+	const [ isFullCartMode, setFullCartMode ] = useState( true );
+
+	const toggleFullCartMode = () => {
+		setFullCartMode( ! isFullCartMode );
+	};
+
+	const getBlockControls = () => {
+		return (
+			<BlockControls className='wc-block-cart-toolbar'>
+				<Toolbar>
+					<TextToolbarButton
+						onClick={ toggleFullCartMode }
+						isToggled={ isFullCartMode }>
+						{  __(
+							'Full Cart',
+							'woo-gutenberg-products-block'
+						) }
+					</TextToolbarButton>
+					<TextToolbarButton
+						onClick={ toggleFullCartMode }
+						isToggled={ ! isFullCartMode }>
+						{  __(
+							'Empty Cart',
+							'woo-gutenberg-products-block'
+						) }
+					</TextToolbarButton>
+				</Toolbar>
+			</BlockControls>
+		);
+	};
+
+	const cart = isFullCartMode ? <FullCart /> : <EmptyCart />;
+
+	return (
+		<Fragment>
+			{ getBlockControls() }
+			{ cart }
+		</Fragment>
+	);
 };
 
 Cart.propTypes = {};
```

Comment:
I believe we don’t do propTypes for blocks? I couldn’t find an example today @Aljullu @nerrad

Reply:
The linter put that there!!

Reply:
We should use `propTypes` for any component.